bin/uid_entrypoint

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if ! whoami &> /dev/null; then
```

Comment:
I would not locate this file in `/bin` folder, otherwise, it will be part of the `npm` package.

Reply:
Mmm, you are right, I didn't realize this. I will move it to a `docker-bin` directory, if you guys have a better approach, let me know.

Reply:
it's fine for me `docker-bin`